services/github_service.py

```diff
@@ -1278,12 +1278,9 @@ def get_current_contributors_for_active_repos(self) -> dict[str, set[str]]:
 
         logins = self.get_org_members_login_names()
         active_repos = [repo.get("name") for repo in self.fetch_all_repositories_in_org()]
-        number_of_repos = len(active_repos)
         active_repos_and_current_contributors = []
         count = 1
         for repo_name in active_repos:
-            print(f"Getting current contributors to {self.organisation_name}/{repo_name}: repo {count} of {number_of_repos}")
```

Comment:
want to keep this print statement - it shows how far you are through all the repos - doesn't print logins